integrations/langchain-js/README.md

```diff
@@ -27,6 +27,22 @@ export BRAINTRUST_API_KEY="your-api-key"
 
 ### Basic Usage
 
+```typescript
+import { ChatOpenAI } from "@langchain/openai";
+import { BraintrustCallbackHandler, init } from "@braintrust/langchain-js";
+
+// Create the callback handler (optionally pass in a custom logger)
+const handler = new BraintrustCallbackHandler();
+
+// Set the handler for all LangChain components
+init({ handler });
```

Comment:
should we call it initHandler? At a first read, I wasn't sure what this does

Reply:
Could we also simplify the signature to take just a handler; the user could call the constructor with the options if necessary?

Reply:
@sachinpad could you expand on just passing the handler with an example?

Reply:
opted with `setGlobalHandler`

Reply:
setGlobalHandler sgtm, though initHandler would be more consistent with other SDK functions (e.g. initLogger).

I'm thinking:
```ts
initHandler(new BraintrustCallbackHandler());
```

If we want to set some opts:
```ts
initHandler(new BraintrustCallbackHandler({ my: opts }));
```